tests/Integration/MatchesSnapshotTest.php

```diff
@@ -18,10 +19,15 @@ public function setUp(): void
         $this->setUpComparesSnapshotFiles();
 
         $updateArgument = array_search('--update-snapshots', $_SERVER['argv']);
+        $noCreateArgument = array_search('--no-create-snapshots', $_SERVER['argv']);
```

Comment:
Can these be renamed?

```suggestion
        $withoutCreatingArgument = array_search('--without-creating-snapshots', $_SERVER['argv']);
```